app/javascript/components/public_health/actions/UpdateAssignedUser.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button, Modal, Form } from 'react-bootstrap';
+import _ from 'lodash';
+import axios from 'axios';
+
+import InfoTooltip from '../../util/InfoTooltip';
+import reportError from '../../util/ReportError';
+
+class UpdateAssignedUser extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      assigned_user: '',
+      initial_assigned_user: undefined,
+      apply_to_household: false,
+      loading: false,
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.submit = this.submit.bind(this);
+  }
+
+  componentDidMount() {
+    const distinctAssignedUser = [...new Set(this.props.patients.map(x => x.assigned_user))];
+
+    var state_updates = {};
+    if (distinctAssignedUser.length === 1 && distinctAssignedUser[0] !== null) {
+      state_updates.initial_assigned_user = distinctAssignedUser[0];
+      state_updates.assigned_user = distinctAssignedUser[0];
+    }
+
+    if (Object.keys(state_updates).length) {
+      this.setState(state_updates);
+    }
+  }
+
+  handleChange(event) {
+    event.persist();
+    if (
+      event.target.id === 'assigned_user_input' &&
+      (event.target.value === '' ||
+        (event.target.value && !isNaN(event.target.value) && parseInt(event.target.value) > 0 && parseInt(event.target.value) <= 9999))
+    ) {
+      this.setState({ assigned_user: event.target.value ? parseInt(event.target.value) : '' });
+    } else if (event.target.id === 'apply_to_household') {
+      this.setState({ apply_to_household: event.target.checked });
+    }
+  }
+
+  submit() {
```

Comment:
Very minor, but we're shifting in the styling for declaring bound functions like this. You can change `submit () {` to `submit = () => {`, and handleChange above to `handleChange = event => {`, and then you can completely remove lines 19 and 20.

Reply:
Thank you for addressing this!